File: app/src/main/java/com/madonasyombua/budgetingbuddydummy/Activity/LoginActivity.java
package com.madonasyombua.budgetingbuddydummy.Activity;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;

import com.madonasyombua.budgetingbuddydummy.R;

public class LoginActivity extends AppCompatActivity {
 TextView signup;
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_login);

       signup =findViewById(R.id.signup);

        signup.setOnClickListener(new View.OnClickListener() {

            @Override
            public void onClick(View v) {
                // Start the Signup activity
                Intent intent = new Intent(getApplicationContext(), SignupActivity.class);
                startActivity(intent);


            }
        });
    }

    @Override
    public void onBackPressed(){

        // Disable going back to the MainActivity
        moveTaskToBack(true);

    }
}

File: app/src/main/java/com/madonasyombua/budgetingbuddydummy/activity/LoginActivity.kt
package com.madonasyombua.budgetingbuddydummy.activity

import android.content.Intent
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView
import com.madonasyombua.budgetingbuddydummy.R

class LoginActivity : AppCompatActivity() {
    internal val signup = findViewById<TextView>(R.id.src_in)
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_login)

        signup.setOnClickListener {
            // Start the Signup activity
            val intent = Intent(applicationContext, SignupActivity::class.java)
            startActivity(intent)
        }
    }

    override fun onBackPressed() {

        // Disable going back to the MainActivity
        moveTaskToBack(true)

    }
}


